src/main/scala/no/ndla/crowdingateway/CrowdinGatewayProperties.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Part of NDLA crowdin_gateway.
+ * Copyright (C) 2017 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.crowdingateway
+
+import no.ndla.crowdingateway.model.domain.{Language, ProjectDefinition}
+
+import scala.util.Properties.envOrNone
+
+
+object CrowdinGatewayProperties {
+  val ApplicationPort = propOrElse("APPLICATION_PORT", "80").toInt
```

Comment:
Ja, den har plaget meg i lengre tid på de andre komponentene.
For å kjøre opp applikasjonen i IntelliJ, så har jeg alltid måtte inn og endre den hardkodede porten fra port 80 til noe annet. Og når man skal committe, så får man plutselig med den.

Det er av convenience for å kunne kjøre opp applikasjon utenfor docker lokalt.